galleries/examples/lines_bars_and_markers/scatter_hist.py

```diff
@@ -15,14 +18,9 @@
 While `.Axes.inset_axes` may be a bit more complex, it allows correct handling
 of main Axes with a fixed aspect ratio.
```

Comment:
```suggestion
```
Optional, but could be left out. That is also explained at the start of the inset_axes example itself.

Reply:
I guess I think it is also useful to explain why there are two methods where they are listed.  A bit of repetition is probably OK.